Guides/Java/Java - Spring.md

```diff
@@ -147,9 +147,9 @@ Et voila, the app is now up and running at `http[s]://APP_NAME.cloudcontrolled.c
 
 
 [Spring Roo petclinic]: http://static.springsource.org/spring-roo/reference/html/intro.html#intro-exploring-sample
-[Database credentials]: Add-on%20credentials
+[Database credentials]: Add-on-credentials
```

Comment:
This needs to be `https://www.cloudcontrol.com/dev-center/guides/java/add-on-credentials-3`